src/platform/plugins/shared/controls/public/index.ts

```diff
@@ -9,41 +9,13 @@
 
 import { ControlsPlugin } from './plugin';
 
-export {
-  controlGroupStateBuilder,
-  type ControlGroupStateBuilder,
-} from './control_group/utils/control_group_state_builder';
+export { ACTION_CLEAR_CONTROL } from './actions/constants';
```

Comment:
Nit. It doesn't seem like any other packages or plugins import this constant. Maybe we can remove the export from here?